### PR TITLE
db relationships for Product, ProductCategorys and ProductReview

### PR DESCRIPTION
**Pull request by icep0ps**

### CODE REVIEW

**Comment by CourtneyFradreck** [comment]
LGTM!!!

**Review by michaeltsuro — COMMENTED** [review]
May you please map all the relationships that your model has with all other models

File: app/Models/Product.php [hunk 1]
```diff
@@ -2,10 +2,23 @@
...
+    {
+        return $this->hasMany(ProductReview::class);
+    }
+    public function category(): BelongsTo
+    {
+        return $this->BelongsTo(ProductCategory::class);
+    }
+    
+    
 }
```
Comment (icep0ps) [inline app/Models/Product.php]: add more relationships

**Comment by icep0ps** [comment]
@michaeltsuro mapped the relationships can you confirm?

**Comment by CourtneyFradreck** [comment]
LGTM!!

**Review by michaeltsuro — APPROVED** [review]

**Pull request merged**